src/views/Conference.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <Page class="league-page-container">
+    <div
+      class="conference-page"
+    >
+      <div
+        id="jitsi"
+      />
+    </div>
+  </Page>
+</template>
+<script>
+// eslint-disable-next-line import/no-extraneous-dependencies
+import JitsiMeetExternalAPI from 'jitsi-iframe-api';
+import Page from '../components/layout/Page.vue';
+import { IS_MOBILE_DEVICE } from '../utils/util';
+
+export default {
+  name: 'Conference',
+  components: {
+    Page,
+  },
```

Comment:
As far as you enabled `props` in the router, you may accept it as:
```suggestion
  },
  props: {
    name: { type: String, required: true },
  },
```
instead of `this.$route.params.room`